spvc/test/run_spirv_cross_tests.py

```diff
@@ -17,13 +17,55 @@
 from multiprocessing import Pool
 import argparse
 import filecmp
+import itertools
 import os
 import re
 import subprocess
 import sys
 import tempfile
 
 
+def log_unexpected_successes(script_args, successes):
+    if not len(successes):
+        log_string = 'Encountered 0 unexpected successes'
+    else:
+        log_string = 'Encountered {} unexpected success(es):\n'.format(
+            len(successes))
+        for success in successes:
+            log_string += '\t{}\n'.format(success)
```

Comment:
In principle this can generate quadratic memory traffic.  But the number of unexpected successes should be smallish, so this ok for now.
Normally I would use a join on the strings to do the concatenation all in one step.